res/controllers/Reloop-Jockey-3-ME-scripts.js

```diff
@@ -0,0 +1,444 @@
+function Jockey3ME() {}
+
+// Variables
+Jockey3ME.EffectLedMeter = 0;
+Jockey3ME.EffectLedMeterValue = 1;
+Jockey3ME.LedMeterShowValue = 1;
+Jockey3ME.LedMeterShowValueTwo = false;
+Jockey3ME.VuMeter = 0;
+Jockey3ME.scratching = [];
+Jockey3ME.hotcueClearVal = 0;
+Jockey3ME.crossfaderScratch = false;
+Jockey3ME.effectSelectTimer = 0;
+Jockey3ME.num_effectsValue = [0,0,0,0];
+Jockey3ME.effectsAvailable = 5; // Sets how many Effects are Loadable
+Jockey3ME.move_beat_value = 4; // Sets how many Beats Jumping when "MOVE" is Turned
```

Comment:
Reloop's manual for the Traktor mapping says this can be adjusted while playing by pressing the encoder and turning it. Can you implement that?